src/sidebar/components/SearchInput.tsx

```diff
@@ -67,12 +67,6 @@ function useSearchKeyboardShortcuts(
 }
```

Comment:
Maybe worth putting the components that are only used by the legacy search implementation in an `old-search` directory or something, to make it clearer what is going on.

Reply:
Yep, I like the idea. That would allow for the new `FilterAnnotationsStatus` component to be named `FilterStatus` without having name collisions with the old one.